packages/botfuel-dialog/src/views/prompt-view.js

```diff
@@ -23,9 +23,9 @@ const View = require('./view');
  * @extends View
  */
 class PromptView extends View {
-  // eslint-disable-next-line require-jsdoc
-  render({ matchedEntities, missingEntities, dialogData }) {
-    logger.debug('render', { matchedEntities, missingEntities, dialogData });
+  /** @inheritDoc */
+  render(userMessage, { matchedEntities, missingEntities, dialogData }) {
```

Comment:
Maybe we could add `userMessage` (or `context` that contains `userMessage`) ?
This would avoid breaking changes (now and later if we want to add more things in `context`):

```javascript
render({ matchedEntities, missingEntities, dialogData, context: { userMessage } }) {
```

Reply:
userMessage replaces userId which was mandatory such it is not optional
I think that it should be the first param for this reason